blockchain/blockchain-wallet-v4-frontend
# feat(brokerage): abstracts brokerage related functionality into it's own data/modal sections starting with ACH

## Description (optional)
This PR contains a lot of migrations of code to brokerage folder including changes of [FWLT-999](https://blockchain.atlassian.net/browse/FWLT-999)

## Testing Steps (optional)
Login to wallet go to settings general, make sure that you can add new linked bank, see details of bank or delete it by clicking on remove button or from bank detiail flyout clicking on remove
